### DISCUSSION TITLE
Flatpak viability?

### DISCUSSION THREAD

**Discussion by PrefersAwkward** [hardinfo2/hardinfo2#67] [Ideas]
How viable do we think a flatpak would be? Not asking that it be done, just curious how feasible that is. This might determine whether some distros, especially atomic, include HardInfo2 as a system package.

**Comment by hwspeedy** [comment]
Why is flatpak even interesting?
https://flatkill.org/

The distro updating is a night mare, but so far I do not see an alternative...

This program is close to the kernel and distro and should be in all distro repos, which is getting closer - the problem is the updating across 100 of distros...

**Reply by PrefersAwkward** [reply]
I'll read that link later today. Just to answer why flatpak is interesting: some atomic distros do not include this package, so we have to overlay it as users. They prefer Flatpak for (almost) anything that's not a 'System' software concept, such as VSCode or Google Chrome.

I hope I'm being fair to every side here. I just wanted this included in Fedora's Atomic distro series, so I thought I'd ask.

**Reply by hwspeedy** [reply]
Yes, I know about the Fedora atomic ideas - it's still 50/50 if that has any chance of a long life. The idea like "android" (immutable) is not that bad, but it is against everything FOSS stands for (freedom/hacking/etc.). Lets see in 5 years...

There must be repo tools like RPM in atomic - then just make a repo and share that.... (Hacking Atomic :)

Sofar I like we have a common FOSS repo database - even though it is not that common...
